src/cells/model.ts

```diff
@@ -429,6 +438,15 @@ class CodeCellModel extends BaseCellModel implements ICodeCellModel {
   }
 
   /**
+   * Clear the cell state.
+   */
+  clear(): void {
+    this.output.clear(false);
+    this.executionCount = null;
+    this.input.prompt = 'In [ ]:';
```

Comment:
Cool, I'm finishing up https://github.com/jupyter/jupyter-js-plugins/pull/154 and associated changes, will come back to this.